addon-test-support/assertions/deprecations-include.js

```diff
@@ -0,0 +1,13 @@
+import { getDeprecations } from '@ember/test-helpers';
+
+export default function deprecationsInclude(expected) {
```

Comment:
should we merge this with `expectDeprecation` and simply support `expecetDeprecation`'s "matcher" being an array of deprecation messages to include?